examples/Advanced_Sounding.py

```diff
@@ -63,7 +61,7 @@
 
 # An example of a slanted line at constant T -- in this case the 0
 # isotherm
-l = skew.ax.axvline(0, color='c', linestyle='--', linewidth=2)
+lcl_pressure = skew.ax.axvline(0, color='c', linestyle='--', linewidth=2)
```

Comment:
Why are we even saving the line that comes out of the plot command? It's not even used later.